addons/rose/addon/components/rose/header/nav/index.hbs

```diff
@@ -5,9 +5,7 @@
 
 <nav class='rose-header-nav' aria-label='header-nav'>
   {{yield
-    (hash
-      link=(component 'link-to' class='rose-header-nav-link')
-      dropdown=(component 'rose/dropdown')
-    )
+    (hash link=(component 'link-to') dropdown=(component 'rose/dropdown'))
```

Comment:
I deleted the class name from here because this was never added to the anchor tag. And it was causing the tests to fail. Look at the anchor tag in the screenshot below with no custom class name. 

<img width="824" alt="Screenshot 2023-08-18 at 10 03 43 AM" src="https://github.com/hashicorp/boundary-ui/assets/15043878/d4464372-ca1e-4465-b107-96ee10b47840">

Reply:
Non-blocking suggestion: Looks like this link component is only used in tests, maybe we remove it entirely?

Reply:
It is used in [application header](https://github.com/hashicorp/boundary-ui/blob/fab566714a4ec4175b9eb1b1d3d26d5ffe66d529/ui/admin/app/templates/application.hbs#L15) topbar

Reply:
But it doesn't look like the link is actually being used in the HeaderNav component that it gets yielded?

Reply:
ah yes, the yielding link isn't being used. Thanks for catching that. I'll update